src/docker-client.ts

```diff
@@ -10,6 +10,7 @@ import { RepoTag } from "./repo-tag";
 
 export type Command = string;
 export type ContainerName = string;
+export type NetworkMode = string
```

Comment:
`NetworkMode = 'bridge' | 'host' | 'overlay'` ?

Reply:
That would be awesome, but it's also possible to create private networks in docker with other names, so the best we can do is to leave it open to almost anything.

Reply:
Ok, it seems I was _partially_ right, but my guess was good enough:
![Screenshot from 2020-01-22 19-13-38](https://user-images.githubusercontent.com/251364/72921344-62580a80-3d4b-11ea-9ed0-789aba5a23c1.png)